applications/FluidDynamicsApplication/custom_elements/embedded_navier_stokes.h

```diff
@@ -220,7 +220,7 @@ class EmbeddedNavierStokes : public NavierStokes<TDim, TNumNodes>
             const unsigned int n_gauss = (rData.pos_int_unit_normals).size();
 
             for (unsigned int i_gauss = 0;  i_gauss < n_gauss; ++i_gauss) {
-                Vector& normal = rData.pos_int_unit_normals[i_gauss];
+                array_1d<double,3>& normal = rData.pos_int_unit_normals[i_gauss];
```

Comment:
lets hope that this is always explicitly specified and not with `auto`
=> #7414